command_kit.gemspec

```diff
@@ -7,10 +7,7 @@ Gem::Specification.new do |gem|
 
   gem.name    = gemspec.fetch('name')
   gem.version = gemspec.fetch('version') do
-                  lib_dir = File.join(File.dirname(__FILE__),'lib')
-                  $LOAD_PATH << lib_dir unless $LOAD_PATH.include?(lib_dir)
-
-                  require 'command_kit/version'
+                  require_relative 'lib/command_kit/version'
```

Comment:
This would work, and bundler does generate gemspecs that use `require_relative` to load the `version.rb` file. Is there any specific reason why `require_relative` is preferred?

Reply:
It's shorter, simpler, more elegant, more high lever, more readable, more in the ruby style spirit. I don't see the advantage of keeping the old way manipulating low level stuff like the loading path.